### ISSUE TITLE
[Usage]: Howto quiet the terminal 'Info' outputs in vllm

### ISSUE THREAD

**Issue by rohitnanda1443** [vllm-project/vllm#5338]
### Your current environment

Ubuntu with cuda 12.1

### How would you like to use vllm

I want to know the switch that can be passed during CLI startup which can quite the terminal output.

My terminal is full of Info outputs from vllm ...

**Comment by youkaichao** [comment]
add a logging config? see `VLLM_LOGGING_CONFIG_PATH` in https://docs.vllm.ai/en/stable/serving/env_vars.html and https://github.com/vllm-project/vllm/blob/388596c91437a51d428a447594e9faec340c29b2/vllm/logger.py#L22 for example.

**Comment by Twilight-Shuxin** [comment]
I managed to disable the outputs through modifying vllm's source code, following the guidance provided here: https://stackoverflow.com/questions/70809900/python-fastapi-health-check-logs. 

In my case, I was using `python -m vllm.entrypoints.api_server --model "modelname"` for serving llm model.

Thus I find the file `vllm.entrypoints.api_server` which is under `/home/user/miniforge3/envs/vllm/lib/python3.12/site-packages/vllm/entrypoints` in my case on my file system.
I added the following code to this file to suppress access logs on generate & health endpoints:
```
import logging

class EndpointFilter(logging.Filter):
    def filter(self, record: logging.LogRecord) -> bool:
        return record.args and len(record.args) >= 3 and record.args[2] != "/health" and record.args[2] != "/generate"

logging.getLogger("uvicorn.access").addFilter(EndpointFilter())
```

It worked! Hope this helps :).
